apps/nextjs-app/src/features/app/blocks/view/grid/GridViewBase.tsx

```diff
@@ -153,15 +154,17 @@ export const GridViewBase: React.FC<IGridViewProps> = (props: IGridViewProps) =>
   } = useGridPrefillingRow(columns);
 
   const { mutate: mutateCreateRecord } = useMutation({
-    mutationFn: () =>
-      createRecords(tableId!, {
-        records: [{ fields: prefillingFieldValueMap! }],
+    mutationFn: () => {
+      const defaultFieldValues = extractDefaultFieldsFromFilters(view?.filter);
+      return createRecords(tableId!, {
```

Comment:
I think it might be better to automatically bring values into the edit form before the user creates the record, rather than adding default data during API calls

Reply:
sure, that seems like a better approach